project-base/kubernetes/deployments/consumer-patches/product-visibility-recalculate.yml

```diff
@@ -0,0 +1,28 @@
+-   op: add
+    path: /spec/template/spec/containers/-
```

Comment:
Why you don't just add this into the `webserver-php-fpm.yml` deployment manifest? Is there a use case where you want to not include this consumer in a specific Kustomize variant?

Reply:
The main idea was to keep configuration of the consumers outside the main `php-fpm` deployment. 
But as we spoke, it should be properly placed in the separate pod (which is not yet possible due to missing external filesystem) and this patch only hides the problem.

Moved directly into deployment in 6a2dbde7a63276a91dd5b65c72fc0a22ab330410

Reply:
The changes to `upgrade-instructions-for-calculation-of-product-visibility-asynchronously.md` were not supposed to be in that commit, but OK.

Thanks for the update - it seems less complicated to me now.